src/librustdoc/html/render.rs

```diff
@@ -1289,8 +1289,9 @@ fn settings(root_path: &str, suffix: &str) -> String {
             .into(),
         ("auto-hide-attributes", "Auto-hide item attributes.", true).into(),
         ("auto-hide-method-docs", "Auto-hide item methods' documentation", false).into(),
-        ("auto-hide-trait-implementations", "Auto-hide trait implementations documentation", true)
+        ("auto-hide-trait-implementations", "Auto-hide trait implementation documentation", true)
             .into(),
+        ("auto-collapse-implementors", "Auto-hide trait implementation items", true).into(),
```

Comment:
This is worded a little confusingly.
```suggestion
        ("auto-collapse-implementors", "Auto-hide the types which implement a trait", true).into(),
```